docs_user/modules/con_comparing-configuration-files-between-deployments.adoc

```diff
@@ -36,38 +39,33 @@ ocp_local_config_path=/tmp/ocp
 connection=local
 ssh_cmd=""
 ----
-
-Os-diff uses `ssh_cmd` to access your {OpenStackPreviousInstaller} host via SSH,
-or the host where your cloud is accessible and the podman/docker binary is installed
-and allowed to interact with the running containers. This option could have a different form:
-
++
+<1> Instructs os-diff to access your {OpenStackPreviousInstaller} host through SSH.
+<2> The host to use to access your cloud, and the podman/docker binary is installed and allowed to interact with the running containers. You can leave this key blank.
++
 ----
 ssh_cmd=ssh -F ssh.config standalone
 director_host=
 ----
-
-----
-ssh_cmd=ssh -F ssh.config
-director_host=standalone
-----
-
++
 or without an SSH config file:
-
++
 ----
 ssh_cmd=ssh -i /home/user/.ssh/id_rsa stack@my.undercloud.local
 director_host=
 ----
-
++
 or
++
 ----
 ssh_cmd=ssh -i /home/user/.ssh/id_rsa stack@
 director_host=my.undercloud.local
 ----
++
+The result of using `ssh_cmd` and `director_host` is a "successful ssh access".
 
-Note that the result of using `ssh_cmd` and `director_host` should be a "successful ssh access".
-
-Configure or generate the `ssh.config` file from inventory or hosts file, for example:
-
+. If you used a host file to connect to your cloud, configure the `ssh.config` file to allow os-diff to access your {OpenStackShort} environment, for example:
++
```

Comment:
Steps 2 and 3 used to be one step, but I split it into 2 steps because I saw this as two different use cases:
1) Using a host file to connect to the cloud
2) Using an inventory file to connect to the cloud
Does that make sense? Can you review both steps and make sure they are accurate?

Reply:
Yes it's true